po/de.po

```diff
@@ -11833,9 +11843,8 @@ msgid "visualize the bisection"
 msgstr "binäre Suche visualisieren"
 
 #: builtin/bisect--helper.c:1273
-#, fuzzy
 msgid "use <cmd>... to automatically bisect"
-msgstr "verwende <Programm>... für die automatische binäre Suche."
+msgstr "verwende <Programm>... für die automatische binäre Suche"
```

Comment:
würde da glaube ich ein Leerschritt vor dem Punkt Punkt Punkt einbauen

Reply:
Das würde ich ungern vom ursprünglichen String abändern wollen. Hab mal geschaut: auch die anderen Übersetzer in anderen Sprachen ändern daran nichts, weil das vielleicht "mehrere Programme" bedeutet (?)...